js/idex.js

```diff
@@ -21,6 +21,7 @@ module.exports = class idex extends Exchange {
             'certified': true,
             'requiresWeb3': true,
             'has': {
+                'associateWallet': true,
```

Comment:
Not a unified method, let's remove this.

Reply:
What makes something unified? If it's on more than one exchange is it unified? 

I also feel like I would rely on has to tell me all the functionalities other than parsers, internal helper methods, ...

Reply:
> If it's on more than one exchange is it unified?

Yes, also, a method is unified if it is implemented across the majority of exchanges, listed in the Manual or in the examples. Some of the unified methods are still not documented though. In this specific case `associateWallet` is idex-only, not unified.

Reply:
If we don't get a dynamic solution for has, this will for sure mean that the first exchange to get a method with have it missing from has

Also, I feel like has should list all the capabilities of an exchange (other than implicit) and not just the unified ones

If I run something like 

```
% ccxt idex has
2021-12-15T05:15:29.108Z
Node.js: v14.17.0
CCXT v1.63.70
{          loadMarkets:  true,
       cancelAllOrders:  undefined,
           cancelOrder:  true,
          cancelOrders:  undefined,
                  CORS:  undefined,
  createDepositAddress:  undefined,
      createLimitOrder:  true,
     createMarketOrder:  true,
           createOrder:  true,
               deposit:  undefined,
             editOrder: "emulated",
          fetchBalance:  true,
         fetchBidsAsks:  undefined,
       fetchBorrowRate:  undefined,
      fetchBorrowRates:  undefined,
     fetchClosedOrders:  true,
       fetchCurrencies:  true,
     ...
```

I want to know everything that's been written for this exchange, not just the unified stuff

Reply:
> I want to know everything that's been written for this exchange, not just the unified stuff

Ok, but you don't need `has` for that, cause `has` is meant to hold the unified methods. If you want to print out a list of all implemented methods for an exchange instance you can do that with code-introspection using the built-in methods of the programming language, so it's a bit out of the scope of the library itself. I mean, in this case, the purpose of `has` is to list the unified methods that are supported for this exchange according to the Manual.

Reply:
I agree, the 'exchange-capabilities' specifically for ccxt (in this context) should be primarily showing what unified methods are implemented for each exchange. In generally-speaking manner to find out the actual 'exchange capabilities' (which will show actually what exchange has what capability) it might be done by passing argument to `exchagne-capabilities.js`. However, main purpose should be just showing unified methods, otherwise that script will loose point and be a huge mixup of individual functions (which are green only on 1 or 2 exchanges) and will not be helpful for us in main purpose.